app/templates/components/pagination-nav.hbs

```diff
@@ -1,3 +1,4 @@
+{{#if chapter.pagination.percentageComplete}}
```

Comment:
This hack will break if percentage starts at 0% rather than the current % that reflects which question the user is rather than how many are behind that question (current minus one).